docs/docs/_partials/footer.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const year = new Date().getFullYear();
+
+export default (
+    <div className="content has-text-centered is-size-5">
+        <p>
+            Copyright © {year} - Built with <a className="is-underlined" href="https://mangelmaxime.github.io/Nacara/" style={{color: "white"}}>Nacara</a>
```

Comment:
We can change this message if you'd like, but I think it's fair to mention the tool used. In the current version we're doing it for Fornax.